### DISCUSSION TITLE
place the picker above the field

### DISCUSSION THREAD

**Discussion by vincenttrr** [easepick/easepick#30] [Q&A]
Hello,
when my picker is open it goes beyond the client window because my input button is located at the bottom of my page. The user cannot validate the calendar. Is there an option to automatically put the picker back in the window? or is it possible to tilt the picker over the field?

**Comment by wakirin** [comment]
Hello,
thanks for the report. I will fix it in the next release.

**Reply by vincenttrr** [reply]
thanks !
